mass_mailing_resend/README.rst

```diff
@@ -0,0 +1,105 @@
+====================
+Resend mass mailings
+====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsocial-lightgray.png?logo=github
+    :target: https://github.com/OCA/social/tree/13.0/mass_mailing_resend
+    :alt: OCA/social
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/social-13-0/social-13-0-mass_mailing_resend
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/205/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+A frequent need for users of mass mailings is to resend one mailing that has
+already been sent in the past to new recipients that haven't received yet that
+mail. But the problem is to know which are the applicable ones.
+
+Odoo already includes a method in its mass mailing logic that avoids to resend
+the same mail 2 times for one mass mailing, and for v9, there was a trick that
+allows to modify the state of a mass mailing from kanban view, covering the
+need.
+
+But now since v10 both status bar in form view and dragging between states in
+kanban are not allowed.
+
+This module introduces a button to restart a mass mailing to draft state,
+allowing you to reevaluate the sending domain or list for performing again
+the mailing.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to *Email marketing > Mailings > Mass Mailings*.
```

Comment:
```suggestion
* Go to *Email marketing > Mailings*.
```
![2020-02-07_10-42-50](https://user-images.githubusercontent.com/8736623/74018734-b3483f80-4996-11ea-9a0b-60f6c188c02c.png)

Reply:
It's already changed on readme/CONFIGURE.rst, but I have regenerated the main README for having it already.